docker/grafana/dashboards/dashboard.json

```diff
@@ -3686,10 +4444,11 @@
                   "type": "fill"
                 }
               ],
+              "hide": false,
               "measurement": "sawtooth_validator.threadpool.InstrumentedThreadPoolExecutor.task_run_time",
               "orderByTime": "ASC",
               "policy": "default",
-              "query": "",
+              "query": "SELECT last(\"99_percentile\") FROM \"sawtooth_validator.Component-threadpool.task_run_time\" WHERE $timeFilter GROUP BY time($__interval), \"host\" fill(none)",
```

Comment:
This is reverting a previous change to the dashboard, where the names were updated and the query field removed.

Reply:
There are also a few more below